### DISCUSSION TITLE
Admin Dashboard Templates

### DISCUSSION THREAD

**Discussion by Purus** [saadeghi/daisyui#1698] [Q&A] [answered]
I am looking for a simple admin dashboard layout and could not find any. The one that find are based on some JS frameworks like React, which is an overkill for me.

Anything simpler with some basic JS, DaisyUI tabs, side menus etc. 

If there is any such example, can you please point me in the right direction.

**Comment by saadeghi** [comment, marked as answer]
daisyUI doesn't provide full page layouts (like admin dashboard, landing page, etc) yet. But it's really easy to make one yourself.  
It would be basically [drawer](https://daisyui.com/components/drawer/) + [navbar](https://daisyui.com/components/navbar/) and the rest depends on the content. You can use [stat](https://daisyui.com/components/stat/), [table](https://daisyui.com/components/table/) some charts, etc

Let me know if you have any questions
